src/components/SortableTable.js

```diff
@@ -150,7 +155,7 @@ class SortableTable extends React.Component {
                   if (this.props.calculatingPage) {
                     return 'page-item disable-button';
                   }
-                  return 'page-item active';
+                  return 'page-item';
```

Comment:
Ana decided to remove the active style from table pagination buttons, so both pagination buttons have the same style now.